core/templates/dev/head/i18n.js

```diff
@@ -75,8 +75,9 @@ oppia.config([
   var availableLanguageKeys = [];
   var availableLanguageKeysMap = {};
   for (var prop in GLOBALS.SUPPORTED_SITE_LANGUAGES) {
-    availableLanguageKeys.push(prop);
-    availableLanguageKeysMap[prop + '*'] = prop;
+    availableLanguageKeys.push(GLOBALS.SUPPORTED_SITE_LANGUAGES[prop].id);
```

Comment:
I don't think this will work. `prop` will be an element in the `GLOBALS.SUPPORTED_SITE_LANGUAGES` list, and can't be used to index `GLOBALS.SUPPORTED_SITE_LANGUAGES`. Instead, you should iterate over the list with `for (var i = 0; i < GLOBALS.SUPPORTED_SITE_LANGUAGES.length; i++)` and access elements as `GLOBALS.SUPPORTED_SITE_LANGUAGES[i].id`.